src/components/common/TmPage.vue

```diff
@@ -22,6 +22,17 @@
         <slot v-else-if="dataEmpty" name="no-data">
           <TmDataEmpty />
         </slot>
+        <slot v-else-if="notFound" name="no-data">
```

Comment:
is there any case where "empty data" would be different then "not found data"? if it isn't I would advice against the additional case.

Reply:
We could just show the dataEmpty page with its default information but we wanted to show the user a link to see all validators when they entered an invalid address. Do you think to keep the code simpler we should just show them the TmDataEmpty page?

Reply:
No the intention was correct. I agree with showing the better guiding info. But I propose that you overwrite the content of the `no-data` slot (https://vuejs.org/v2/guide/components-slots.html#Fallback-Content) on the `PageValidator` instead of defining this domain specific message in the lower level component `TmPage`.

Reply:
I see what you are saying and will make the changes. Thanks